mods/farming/api.lua

```diff
@@ -319,11 +319,9 @@ farming.register_plant = function(name, def)
 		nodegroups[pname] = i
 
 		local next_plant = nil
-		local on_timer = nil
 
 		if i < def.steps then
 			next_plant = mname .. ":" .. pname .. "_" .. (i + 1)
-			on_timer = farming.grow_plant
```

Comment:
I'm pretty sure it's an oversight - while I heavily debugged the code, assignments like this to locals that are not used are hard to spot.